README.md

```diff
@@ -1,65 +1,93 @@
 
-Streamlit makes it incredibly easy to build interactive apps:
+Read more on getting started with Streamlit in our [documentation](https://docs.streamlit.io/library/get-started).
 
-```python
-import streamlit as st
 
-x = st.slider('Select a value')
-st.write(x, 'squared is', x * x)
-```
+## Quick start
 
-<img src="https://raw.githubusercontent.com/streamlit/docs/main/public/images/simple_example.png"/>
+#### A little example
 
-## A bigger example
-
-Streamlit's simple and focused API lets you build incredibly rich and powerful tools.  [This demo project](https://github.com/streamlit/demo-self-driving) lets you browse the entire [Udacity self-driving-car dataset](https://github.com/udacity/self-driving-car) and run inference in real-time using the [YOLO object detection net](https://pjreddie.com/darknet/yolo).
-
-![Final App Animation](https://raw.githubusercontent.com/streamlit/docs/main/public/images/complex_app_example.gif)
-
-The complete demo is implemented in less than 300 lines of Python. In fact, the app contains [only 23 Streamlit calls](https://github.com/streamlit/demo-self-driving/blob/master/streamlit_app.py) which illustrates all the major building blocks of Streamlit. You can try it right now at [share.streamlit.io/streamlit/demo-self-driving](https://share.streamlit.io/streamlit/demo-self-driving).
+Streamlit makes it incredibly easy to build interactive apps:
 
-## The Streamlit GitHub badge
+```python
+import streamlit as st
+x = st.slider("Select a value")
+st.write(x, "squared is", x * x)
+```
 
-Streamlit's GitHub badge helps others find and play with your Streamlit app.
+<img src="https://user-images.githubusercontent.com/7164864/214900507-1c89fc6b-e196-4f5c-890f-7e0be62d5d9d.png" width=300 alt="Little example"></img>
+
+#### Give me more!
+
+Streamlit comes in with [a ton of additional powerful elements](https://docs.streamlit.io/library/api-reference) to spice up your data apps and delight your viewers. Some examples:
+
+<table border="0">
+   <tr>
+     <td><img src="https://docs.streamlit.io/images/api/date_input.jpg" style="max-height:150; width:auto; display:block;"></td>
+     <td><img src="https://user-images.githubusercontent.com/7164864/215110064-5eb4e294-8f30-4933-9563-0275230e52b5.gif"  style="max-height:150; width:auto; display:block;"></td>
+     <td><img src="https://docs.streamlit.io/images/api/line_chart.jpg" style="max-height:150; width:auto; display:block;"></td>
+     <td><img src="https://docs.streamlit.io/images/api/tabs.jpg" style="max-height:150; width:auto; display:block;"></td>
+     <td><img src="https://docs.streamlit.io/images/mpa-add-pages.png" style="max-height:150; width:auto; display:block;"></td>
```

Comment:
The multipage app doesn't really show what that is, I think mostly because it's so big and the actual multipage content is so small. It also seems a bit out of place since it has a browser window but all the other elements don't. Maybe we zoom in on just the page navigation and show that without the window around it?

Reply:
Added 
![CleanShot 2023-01-27 at 20 06 47](https://user-images.githubusercontent.com/7164864/215174643-2ac63564-d1a2-4191-bde0-c8e119daa3f1.gif)